spec/fixtures/mensual_sample.rb

```diff
@@ -0,0 +1,132 @@
+# Mensual 
+# FTF_B88353412_202501_001.txt
+
+{
+  "fiscal_id": "B88353412",
+  "period": "202501",
+  "request_id": "",
+  "file_type": "M",
+  "products": [
+    # Data register 10 (product) ADD
+    {
+      "action": "A",
+      "type": "01", # cuenta corriente
```

Comment:
We will probably never have other products, but I'm okay with passing them explicitly as they leave the door open.